network-api/networkapi/wagtailpages/models.py

```diff
@@ -786,10 +786,6 @@ class ParticipatePage2(PrimaryPage):
     ]
 
 
-class PeoplePage(PrimaryPage):
```

Comment:
This bit and its associated migration needs to be done as a separate PR. First remove all references to the model, land that, then a second PR to do the actual class removal.